lib/containers/urls.pm

```diff
@@ -52,64 +52,207 @@ sub get_opensuse_registry_prefix {
     }
 }
 
+our %registries = (
```

Comment:
```suggestion
our %images = (
```
I'm sorry, but we're not listing **registries** there. We're listing image names.

Reply:
https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/13141#discussion_r705007863

in brief: `registry` is the location that an image is stored. the *image* is defined by the whole uri (local: locahost/alpine, remote registry.whatever.com/alipine). We cant name everything *images* and i thing *%registries* solves that.

Reply:
As I said, I don't care about the name, but answering to this comment, you are actually providing the full uri of the images, e.g. `registry.suse.de/suse/sle-12-sp3/docker/update/cr/totest/images/suse/sles12sp3`, you are not providing just the registry `registry.suse.de`.